catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -258,8 +274,8 @@ private boolean orderContainsSupportedDelivery() {
         if (order.del_list != null) {
             for (DeliveryDetails deliveryDetails : order.del_list) {
                 Destination destination = deliveryDetails.dests;
-                if (destination.discriminator() == DestinationType.FTP
-                        && destination.f_dest() != null) {
+                if (((destination.discriminator() == DestinationType.FTP) && (destination.f_dest() != null)) ||
+                        ((destination.discriminator() == DestinationType.EMAIL) && (destination.e_dest() != null))) {
```

Comment:
This if-condition is getting pretty hard to read. Might want to break is a part into several is-methods so that the code reads more naturally.